src/visitor.rs

```diff
@@ -640,10 +659,7 @@ impl<'b, 'a: 'b> FmtVisitor<'a> {
                         ErrorKind::DeprecatedAttr,
                     )],
                 );
-            } else if attr.path.segments[0].ident.to_string() == "rustfmt"
-                && (attr.path.segments.len() == 1
-                    || attr.path.segments[1].ident.to_string() != "skip")
-            {
+            } else if self.is_known_rustfmt_attr(&attr.path.segments) {
```

Comment:
@rchaser53 :bowing_man: Should be `unknown` because it checks for unknown attributes within the `rustfmt` namespace.

```suggestion
            } else if self.is_unknown_rustfmt_attr(&attr.path.segments) {
```

Reply:
Oh. I didn't realized it. I Renamed it.

Reply:
Thanks, sorry for the back and forth on that one